### PR TITLE
Django core cache

### PR DESCRIPTION
**Pull request by UnknownPlatypus**
Some misc stubs update.

This removes 12 entries from the todolist

### CODE REVIEW

File: django-stubs/core/cache/backends/filebased.pyi [hunk 1]
```diff
@@ -1,7 +1,8 @@
-from typing import Any
+from typing import Any, ClassVar
 
 from django.core.cache.backends.base import BaseCache
 
 class FileBasedCache(BaseCache):
     cache_suffix: str
```
Comment (sobolevn) [inline django-stubs/core/cache/backends/filebased.pyi]: It is also `ClassVar`

File: django-stubs/core/cache/backends/memcached.pyi [hunk 2]
```diff
@@ -12,6 +12,8 @@ class BaseMemcachedCache(BaseCache):
         library: ModuleType,
         value_not_found_exception: type[BaseException],
     ) -> None: ...
+    @property
+    def client_servers(self) -> Sequence[str]: ...
```
Comment (sobolevn) [inline django-stubs/core/cache/backends/memcached.pyi:16]: Return types should be exact, no need for abstactions (unless explicitly documented).

**Review by sobolevn — COMMENTED** [review]
Thanks!

File: django-stubs/core/cache/backends/filebased.pyi (hunk 1 above)
Comment (UnknownPlatypus) [reply django-stubs/core/cache/backends/filebased.pyi]: indeeed! updated

File: django-stubs/core/cache/backends/memcached.pyi (hunk 2 above)
Comment (UnknownPlatypus) [reply django-stubs/core/cache/backends/memcached.pyi:16]: I thinks it's still correct here.
`self._servers` is set in the init this way:
```python
        if isinstance(server, str):
            self._servers = re.split("[;,]", server)
        else:
            self._servers = server
```

And server is annotated as `server: str | Sequence[str]`. So in the else branch, we could have 
a Sequence. `PyMemcacheCache(("aa", "bb"), {})` is valid for ex.

However, in the `PyLibMCCache` override, it always returns `list[str]`.

I don't know if this distinction is worth it tho. I've updated the PR to match runtime types, let me know what you think

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**